template/haskell-http/src/Server.hs

```diff
@@ -45,5 +46,5 @@ getEnvironment varname defvalue = (varname,) . fromMaybe defvalue <$> lookupEnv
 envList :: [(String, String)]
 envList = [("HOSTNAME", "localhost"), ("TEST","DEFAULT")]
 
-getContext :: IO [(String, String)]
-getContext = mapM (uncurry getEnvironment) envList
\ No newline at end of file
+--getContext :: IO [(String, String)]
+--getContext = mapM (uncurry getEnvironment) envList
```

Comment:
It's ok to remove the commented out code
https://kentcdodds.com/blog/please-dont-commit-commented-out-code